tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh

```diff
@@ -49,6 +49,7 @@ bazel test \
       --local_test_jobs=${N_TEST_JOBS} \
       --test_env=TF_GPU_COUNT=$TF_GPU_COUNT \
       --test_env=TF_TESTS_PER_GPU=$TF_TESTS_PER_GPU \
+      --test_env=TF_PER_DEVICE_MEMORY_LIMIT_MB=1920 \
```

Comment:
why the downgrade to 1920?
(default is 2048 - set here - https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/tools/ci_build/gpu_build/parallel_gpu_execute.sh#L27 )

Reply:
Since the change to rocm_driver and Dockerfile.rocm moved the default memory reserve out of docker and into TF, I'm seeing failures in the unit test //tensorflow/compiler/xrt/tests:raw_api_test_gpu subtest RawApiTest.TestMemoryInfo (which I can't reproduce locally). 
I thought dropping to 1920 would resolve these failures, but it didn't.